packages/angular_devkit/schematics/tools/file-system-engine-host.ts

```diff
@@ -36,6 +36,17 @@ export class FileSystemEngineHost extends FileSystemEngineHostBase {
       return join(this._root, name, 'collection.json');
     }
 
+    // Allow `${_root}/${name_without_@schematics_prefix}/collection.json.
+    const SCHEMATICS_PREFIX = '@schematics/';
```

Comment:
I don't follow the logic in this. If it starts with `@schematics/` try without the prefix? Why not change the default collection instead?

Reply:
If you think this is too much hack here, we could change the default collection instead. This is just in case we might use external schematics internally some day and external schematics can still resolve its dependent collection without local modification.